tests/functional/ui/test_odf_storage_consumption_trend.py

```diff
@@ -0,0 +1,314 @@
+    To accommodate this, a tolerance of 10% difference between the manual and UI values is allowed.
+
+    For example, if the manual calculation yields a value of 10 and the UI shows 11 as the average,
+    the test will still pass.
+    However, if the UI value is 12 or greater, the test will fail due to exceeding the 10% tolerance.
+
+    """
+
+    def get_est_days_from_ui(self):
+        """
+        Get the value of 'Estimated days until full' from the UI
+
+        Returns:
+            est_days: (int)
+
+        """
+        validation_ui_obj = ValidationUI()
+        collected_tpl_of_days_and_avg = (
+            validation_ui_obj.odf_storagesystems_consumption_trend()
+        )
+        est_days = re.search(r"\d+", collected_tpl_of_days_and_avg[0]).group()
+        logger.info(f"'Estimated days until full' from the UI : {est_days}")
+        return int(est_days)
+
+    def get_avg_consumption_from_ui(self):
+        """
+        Get the value of 'Average storage consumption' from the UI
+
+        Returns:
+            average: (float)
+
+        """
+        validation_ui_obj = ValidationUI()
+        collected_tpl_of_days_and_avg = (
+            validation_ui_obj.odf_storagesystems_consumption_trend()
+        )
+        average = float(
+            re.search(r"-?\d+\.*\d*", collected_tpl_of_days_and_avg[1]).group()
+        )
+        logger.info(f"'Average of storage consumption per day' from the UI : {average}")
+        return average
+
+    def calculate_est_days_and_average_manually(self):
+        """
+        Calculates the 'Estimated days until full' manually by:
+        1. Get the age of the cluster in days
+        2. Get used capacity of the cluster
+        3. Get total capacity of the cluster
+        4. Calculate average consumption of the storage per day
+        5. Calculate the 'Estimated days until full' by using average and available capacity.
+
+        Returns:
+            estimated_days_calculated: (tuple)
+
+        """
+        number_of_days = get_age_of_cluster_in_days()
+        logger.info(f"Age of the cluster in days: {number_of_days}")
+        tpl_of_used_and_total_capacity = get_used_and_total_capacity_in_gibibytes()
+        used_capacity = tpl_of_used_and_total_capacity[0]
+        logger.info(f"The used capacity from the cluster is: {used_capacity}")
+        total_capacity = tpl_of_used_and_total_capacity[1]
+        available_capacity = total_capacity - used_capacity
+        logger.info(f"The available capacity from the cluster is: {available_capacity}")
+        average = used_capacity / number_of_days
+        logger.info(f"Average of storage consumption per day: {average}")
+        estimated_days_calculated = available_capacity / average
+        manual_est = math.floor(estimated_days_calculated)
+        logger.info(f"Estimated days calculated are {manual_est}")
+        return (manual_est, average)
+
+    def test_consumption_trend_card_ui(self, setup_ui_class):
```

Comment:
pls add polarion id for every test

Reply:
I think this test may be removed, so we can save run time and analysis efforts.
information from this card is gathered and validated in the tests bellow

Reply:
This is 4.17 test case. Polarion id will be added once the tests pushed into polarion.

This is critical test case which ensures that the basic widget is present or not with all required tabs [text information]. This test cannot be removed.

Reply:
management requested us to not merge tests without polarion ID because we don't have a mechanism to check-up if all tests are with Polarion-ids. Then, if no Polarion-id we'll not get statistics/results with regression.
Same, it will make difficult to a black squad to analyze results, and my do a mess in a Regression spreadsheet. 
 
Engineers usually add ids into Polarion manually and update Test plan spreadsheet. Then the ids will be pulled/updated to Polarion with a script, if thest is not automated yet, then id will be assigned. You also will have to create a Polarion requirement as a part of test development and documentation process. 
@ramkiperiy, please correct me with process if I am wrong.

Reply:
I agree with @DanielOsypenko 
@nagendra202 Please create test case(s) in polarion and add the polarion id for every test.

Reply:
done